app.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
I'm curious as to what this file does. Could you fill me in?
It seems like it's related to the Heroku story. If so, do you mind removing it from this PR and adding it to the Heroku one since it's out of the scope of work for this PR.

Reply:
Yes. The file is required in order to create a review app. Without it, a review app will not be created.

Reply:
My understanding is that when the heroku task is merged in (it actually has been merged in), it should add this file. Then you update your branch with the develop branch and this branch should also have the file. My issue with this is that it's outside the scope of work. Also, there's a different PR that's done this for us.
Would love to hear your thoughts on this

Reply:
After doing some research, I discovered that in most cases, this file is added to the develop branch and then when subsequent branches are created from develop, they will, of course, contain this file. I was more inclined to have it here in this branch so I can see whether the branch works or not (since when I was branching out of develop, it did not have the app.json) but considering your valid concern to constrain this PR to the scope of work in the story, I should remove it.